helix-term/src/ui/editor.rs

```diff
@@ -569,9 +572,22 @@ impl EditorView {
         cxt: &mut commands::Context,
         event: KeyEvent,
     ) -> Option<KeymapResult> {
+        self.autoinfo = None;
         match self.keymaps.get_mut(&mode).unwrap().get(event) {
             KeymapResult::Matched(command) => command.execute(cxt),
-            KeymapResult::Pending(_map) => { /* render infobox */ }
+            KeymapResult::Pending(map) => {
+                let mut body = BTreeMap::new();
+                for (key, trie) in map.into_iter() {
+                    let desc = match trie {
+                        KeyTrie::Leaf(cmd) => cmd.doc(),
+                        KeyTrie::Node(_) => "",
```

Comment:
We also need to have doc for node, it will be shown as the title.

Reply:
Fixed 👍🏾